quantus/metrics/randomisation/model_parameter_randomisation.py

```diff
@@ -438,3 +389,6 @@ def compute_correlation_per_sample(
         corr_coeffs = list(results.values())
 
         return corr_coeffs
+
+    def evaluate_batch(self, *args, **kwargs):
+        raise RuntimeError("This is unexpected.")
```

Comment:
why "unexpected", please write a bit more elaborate error message!

Reply:
Changed to `"`evaluate_batch` must never be called for `ModelParameterRandomisation`."`